### ISSUE TITLE
calc_anchors/trained yolov3-tiny problem

### ISSUE THREAD

**Issue by ghasemabdi** [AlexeyAB/darknet#3502]
Hello all: I tried to estimate the anchors by this command "darknet.exe detector calc_anchors data/obj.data -num_of_clusters 6 -width 416 -height 416"; the command was run but without any computation and estimated information. on the other hand, I trained the yolov3-tiny (via the original anchors and this command "darknet.exe detector train data/obj.data yolov3-tiny-obj.cfg yolov3-tiny.conv.15") on my custom dataset until my avg loss reached to 0.001 and surprisingly found that the trained net could not detect anything (by this command "darknet.exe detector test data/obj.data yolov3-tiny-obj.cfg yolov3-tiny-obj_xxx.weights.weights"). Could you please advise me within these questions? Best, Ghasem

![image](https://user-images.githubusercontent.com/52261703/60219972-bb30f480-9889-11e9-819c-739e2bd009eb.png)

My "obj.data" is as below:
classes  = 1
train     = data/train.txt
valid     = data/test.txt
names  = data/obj.names
backup = backup/

My "yolov3-tiny-obj.cfg" is as below:
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=8
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 2000
policy=steps
steps=1600,1800
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

**Comment by ghasemabdi** [comment]
@AlexeyAB
